ui/run.sh

```diff
@@ -118,10 +118,11 @@ case "$(basename ${0})" in
         run ecsdeploy check || exit $?
         run ecsdeploy bootstrap || exit $?
         run ecsdeploy reboot || exit $?
-        sleep 10
+        sleep 30
         run ping_until_clear
         run ecsdeploy deploy || exit $?
         run ecsdeploy start || exit $?
+	o "Please wait for 30 minutes before running next step for services to bootstrap"
```

Comment:
Use the correct API calls to determine cluster readiness, not customer behavior.

Reply:
This is the end of step1 , either we should monitor the service readiness before exit or implement a service readiness check on step2.   can we move this to 3.6.2 ?